plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/converter/GlueToTrinoConverter.java

```diff
@@ -159,7 +159,10 @@ public static Table convertTable(com.amazonaws.services.glue.model.Table glueTab
             if (sd == null) {
                 throw new TrinoException(HIVE_UNSUPPORTED_FORMAT, "Table StorageDescriptor is null for table '%s' %s".formatted(table, glueTable));
             }
-            boolean isCsv = sd.getSerdeInfo() != null && HiveStorageFormat.CSV.getSerde().equals(sd.getSerdeInfo().getSerializationLibrary());
```

Comment:
Why NPE happens even if we have `sd.getSerdeInfo() != null` condition?

Reply:
It happened in `setStorageBuilder` later on.